examples/jquery/bower.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "todomvc-jquery",
   "dependencies": {
-    "todomvc-common": "~0.3.0",
+    "todomvc-common": "~1.0.1",
+    "todomvc-app-css": "~1.0.1",
     "jquery": "~2.1.1",
     "handlebars": "~2.0.0",
     "director": "~1.2.2"
```

Comment:
Yes, I know. Not a requirement, but nice to do it when we change around things anyways. Thanks for doing it btw :)